### PR TITLE
Improve CMS API docs

### PR DESCRIPTION
**Pull request by sergioifg94**
**What this PR does / why we need it**:

Improve CMS API docs: Fix typos and update summary of operations in order to align with current styling

**Verification steps** 

image

### CODE REVIEW

File: doc/active_docs/CMS API.json [hunk 1]
```diff
@@ -6,8 +6,9 @@
       "operations": [
         {
           "httpMethod": "GET",
-          "summary": "List all files",
+          "summary": "File List",
```
Comment (guicassolato) [inline doc/active_docs/CMS API.json:9]: We usually do not touch these files directly, but change instead in https://github.com/3scale/porta/blob/master/app/controllers/admin/api/cms/files_controller.rb#L37

...and then:

```
doc:swagger:generate:cms
```

...which generates https://github.com/3scale/porta/blob/427633bf033c58076987f04d75e0b525cbe9c35f/doc/active_docs/cms.json#L9.

Some minor details might differ on the base branch you are working on (i.e. proper `docs/active_docs/CMS API.json` file now), but the overall flow to generate the docs remains the same.

**Review by guicassolato — CHANGES_REQUESTED** [review]

File: doc/active_docs/CMS API.json (hunk 1 above)
Comment (sergioifg94) [reply doc/active_docs/CMS API.json:12]: Thanks for pointing that our @guicassolato I updated the PR with the changes in the controllers to generate the equivalent documentation, using `source2swagger` in a similar way as the other controllers are documented

**Review by akostadinov — DISMISSED** [review]
Looks awesome.

**Comment by sergioifg94** [comment]
@akostadinov I updated the PR to base on master as we discussed yesterday. Also verified that the documentation is still not customer facing

**Comment by akostadinov** [comment]
incorporated into #2667, will merge together with rails branch so closing this one. Thanks a lot!

**Pull request closed without merging**